lib/Driver/ParseableOutput.cpp

```diff
@@ -201,15 +201,17 @@ class FinishedMessage : public TaskOutputMessage {
 
 class SignalledMessage : public TaskOutputMessage {
   std::string ErrorMsg;
+  Optional<int> Signal;
 public:
   SignalledMessage(const Job &Cmd, ProcessId Pid, StringRef Output,
-                   StringRef ErrorMsg) : TaskOutputMessage("signalled", Cmd,
+                   StringRef ErrorMsg, Optional<int> Signal) : TaskOutputMessage("signalled", Cmd,
```

Comment:
Nit: with the additional parameter, the alignment of the two lines that follow are totally out of whack.